static/autocomplete.html

```diff
@@ -23,7 +23,7 @@
 
         $("input").on("keyup", function() {
             $.ajax({
-                    url: 'http://localhost:900/api/search/' + this.value,
+                    url: 'http://localhost:8080/api/search/' + this.value,
```

Comment:
Same kinda thing here.  You should be able to use something like this (I can't say for sure):
```javascript
url: '/api/search/' + this.value,
```